cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup canola oil
+ - 1/2 cup canola oil (diffirent-oil branch)
```

Comment:
I think we need to remove the comment stating that (different-oil branch) as it is already mentioned in the interface.